nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs

```diff
@@ -992,6 +1002,7 @@ private void ResponseHackAndSlash(ActionBase.ActionEvaluation<HackAndSlash> eval
                             {
                                 UpdateCurrentAvatarStateAsync(eval).Forget();
                                 UpdateWeeklyArenaState(eval);
+                                UpdateHackAndSlashBuffState(eval);
```

Comment:
If the `UpdateHackAndSlashBuffState()` function will be async function,

```suggestion
                                UpdateHackAndSlashBuffStateAsync(eval).Forget();
```

or

```suggestion
private UniTaskVoid ResponseHackAndSlashAsync(eval)
{
                                await UpdateHackAndSlashBuffStateAsync(eval);
}
```